src/ralph/lib/transitions/views.py

```diff
@@ -291,3 +293,11 @@ def get_context_data(self, *args, **kwargs):
             context['are_jobs_running'] = any([j.is_running for j in jobs])
             context['for_many_objects'] = True
         return context
+
+
+class KillTransitionJobView(View, metaclass=PermissionViewMetaClass):
+
+    def get(self, request, *args, **kwargs):
+        job = TransitionJob.objects.get(pk=kwargs.get('job_id'))
+        job.kill()
+        return HttpResponseRedirect(request.META.get('HTTP_REFERER'))
```

Comment:
Not because I wants go back to page (for example):
data_center/datacenterasset/719/current_transitions/
back_office/datacenterasset/719/current_transitions/
or
/async-transitions-awaiter/